Echo/EchoHelp.cpp

```diff
@@ -0,0 +1,85 @@
+#include "PolySyncDataModel.hpp"
+#include "EchoHelp.hpp"
+
+using namespace std;
+
+namespace polysync
+{
+
+EchoHelp::EchoHelp()
```

Comment:
Probably don't need to define or prototype a constructor if it doesn't do anything.